src/test/java/seedu/address/testutil/ListingBuilder.java

```diff
@@ -0,0 +1,74 @@
+package seedu.address.testutil;
+
+import java.util.ArrayList;
+
+import seedu.address.model.listing.Description;
+import seedu.address.model.listing.Listing;
+import seedu.address.model.listing.Title;
+
+/**
+ * A utility class to help with building Person objects.
+ */
+public class ListingBuilder {
+
+    public static final String DEFAULT_TITLE = "This is a default job title";
+    public static final String DEFAULT_DESCRIPTION =
+            "This is a default job description! "
+            + "In this job you are expected to do these things: thing1, thing2 and thing3.";
+    public static final ArrayList<String> DEFAULT_APPLICANTS = new ArrayList<>();
+
+    private Title title;
+    private Description description;
+    private ArrayList<String> applicants;
+
+    /**
+     * Creates a {@code PersonBuilder} with the default details.
```

Comment:
Change to {@code ListingBuilder}

Reply:
updated all PersonBuilder instances to ListingBuilder